src/workerd/jsg/dom-exception.c++

```diff
@@ -49,4 +50,20 @@ void DOMException::visitForGc(GcVisitor& visitor) {
   visitor.visit(errorForStack);
 }
 
+void DOMException::serialize(jsg::Lock& js, jsg::Serializer& serializer) {
+  serializer.writeLengthDelimited(message);
+  serializer.writeLengthDelimited(name);
+  serializer.write(js, JsValue(errorForStack.getHandle(js)));
```

Comment:
Note that `errorForStack` contains a copy of `message`, so that string will be duplicated here. Should we avoid that? Maybe don't bother writing `message` separately, just have it come from `errorForStack`?

Reply:
Well, not sure it matters too much. In the coming weeks I'm going to be pursuing the contribution to v8 that will eliminate the need to have `errorForStack` and attach the `stack` property directly to `DOMException`'s JS wrapper, in which case we'd want to preserve `message`. For now, I think we should just keep this as is and we can tweak later.